src/ImageLoaders/MzExe/Borland/SymbolLoader.cs

```diff
@@ -589,18 +589,71 @@ private void LoadTypeTable(LeImageReader rdr, string[] names)
                 case TID_BOOL: bt = ClassifyType(rdr, PrimitiveType_v1.Bool()); break;
                 case TID_TBYTE: bt = ClassifyType(rdr, new PrimitiveType_v1 { Domain = Core.Types.Domain.Any, ByteSize = 10 }); break;
                 case TID_PWORD: bt = ClassifyType(rdr, new PrimitiveType_v1 { Domain = Core.Types.Domain.Any, ByteSize = 6 }); break;
+                
                 case TID_NEAR: bt = ClassifyComplex(rdr, t => PointerType_v1.Create(t, 2), "near *"); break;
                 case TID_FAR: bt = ClassifyComplex(rdr, t => PointerType_v1.Create(t, 4), "far *"); break;
+           
                 case TID_CARRAY: bt = ClassifyComplex(rdr, t => new ArrayType_v1 { ElementType = t, Length = 1 /*//$TODO: unknown length */}, "C Array[]"); break;
+                case TID_PARRAY:
+                    {
+                        ++i;
+                        var filler = rdr.ReadByte();
+                        var elementType = rdr.ReadLeUInt16();
+                        var arrayIndexType = rdr.ReadLeUInt16();
+                        var f1 = rdr.ReadLeUInt16();
+                        var f2 = rdr.ReadLeUInt16();
+                        var f3 = rdr.ReadLeUInt16();
+                        SerializedType indexDataType;
```

Comment:
Are f1, f2, and f3 filler? Do you think you could label them "filler1", "filler2", and "filler3" to make it clearer? Alternatively, just throw the values away.

Reply:
Sure, renamed them